embassy-stm32/src/lib.rs

```diff
@@ -67,6 +67,10 @@ pub mod wdg;
 #[cfg(feature = "subghz")]
 pub mod subghz;
 
+/// Provides access to unique device ID and flash size in kb
```

Comment:
Does it really provide the flash size after latest updates?

Reply:
No it doesn't I will change that thank you